src/infection-model/index.tsx

```diff
@@ -1,69 +1,93 @@
+    ageGroupPopulations[ageGroupIndex.age45] = age45Population;
+    ageGroupPopulations[ageGroupIndex.age55] = age55Population;
+    ageGroupPopulations[ageGroupIndex.age65] = age65Population;
+    ageGroupPopulations[ageGroupIndex.age75] = age75Population;
+    ageGroupPopulations[ageGroupIndex.age85] = age85Population;
+    ageGroupPopulations[ageGroupIndex.ageUnknown] = ageUnknownPopulation;
+    ageGroupPopulations[ageGroupIndex.staff] = staffPopulation;
+
+    ageGroupInitiallyInfected[ageGroupIndex.age0] = age0Cases;
+    ageGroupInitiallyInfected[ageGroupIndex.age20] = age20Cases;
+    ageGroupInitiallyInfected[ageGroupIndex.age45] = age45Cases;
+    ageGroupInitiallyInfected[ageGroupIndex.age55] = age55Cases;
+    ageGroupInitiallyInfected[ageGroupIndex.age65] = age65Cases;
+    ageGroupInitiallyInfected[ageGroupIndex.age75] = age75Cases;
+    ageGroupInitiallyInfected[ageGroupIndex.age85] = age85Cases;
+    ageGroupInitiallyInfected[ageGroupIndex.ageUnknown] = ageUnknownCases;
+    ageGroupInitiallyInfected[ageGroupIndex.staff] = staffCases;
   } else {
     ageGroupPopulations[ageGroupIndex.ageUnknown] = totalIncarcerated;
+    ageGroupInitiallyInfected[ageGroupIndex.ageUnknown] = confirmedCases;
   }
 
   const curveData = getCurveProjections({
     ageGroupPopulations,
-    initiallyInfected: confirmedCases || 0,
+    ageGroupInitiallyInfected,
+    facilityDormitoryPct,
+    facilityOccupancyPct,
     numDays,
     rateOfSpreadFactor,
   });
+
+  // for these curves we combine incarcerated and staff
+  function combinePopulations(columnIndex: number) {
+    return zip(
+      getAllValues(getColView(curveData.incarcerated, columnIndex)),
+      getAllValues(getColView(curveData.staff, columnIndex)),
+    ).map(([incarcerated, staff]) => incarcerated + staff);
+  }
+
   return {
-    exposed: getAllValues(getColView(curveData, seirIndex.exposed)),
-    infectious: getAllValues(getColView(curveData, seirIndex.infectious)),
-    hospitalized: getAllValues(getColView(curveData, seirIndex.hospitalized)),
+    exposed: combinePopulations(seirIndex.exposed),
+    infectious: combinePopulations(seirIndex.infectious),
+    hospitalized: combinePopulations(seirIndex.hospitalized),
   };
 }
 
-export function estimatePeakHospitalUse(inputs: {
-  incarceratedPopulation: number;
-  R0: number;
-  hospitalBedCapacity: number;
-}) {
-  const { incarceratedPopulation, R0, hospitalBedCapacity } = inputs;
-
-  const curves = calculateCurves({
-    usePopulationSubsets: false,
-    totalIncarcerated: incarceratedPopulation,
-    rateOfSpreadFactor: R0,
-  });
-  const bedsRequiredPerDay = curves.hospitalized;
-  const peakRequirement = Math.max(...bedsRequiredPerDay);
-  const peakUtilization = peakRequirement / hospitalBedCapacity;
-  const peakDay = bedsRequiredPerDay.indexOf(peakRequirement) + 1;
-  return { peakUtilization, peakDay };
+export function estimatePeakHospitalUse() {
+  // TODO: this function is now broken by the latest model updates
```

Comment:
Just for my own knowledge, briefly: why does this and the spinner get broken by the latest model updates?

Reply:
the short answer is that the slider produces a single R0 value, whereas the new model now takes a qualitative "low/moderate/high" output and maps that to two separate R0 values for cells vs dorms.

Reply:
Is there no reasonable way to consolidate R0 values? I'm sure you discussed this with @sychang already, so just trying to understand if there's any quick solution to keep this around for launch.

Reply:
based on my conversation with Serena it didn't really seem worth figuring that out before getting to the rest of the stuff on our punchlist ... there is definitely something to do there it's just less of a priority than other blockers